archivebox/config/__init__.py

```diff
@@ -232,7 +232,7 @@ def get_real_name(key: str) -> str:
     'LOGS_DIR':                 {'default': lambda c: c['OUTPUT_DIR'] / LOGS_DIR_NAME},
     'CONFIG_FILE':              {'default': lambda c: Path(c['CONFIG_FILE']).resolve() if c['CONFIG_FILE'] else c['OUTPUT_DIR'] / CONFIG_FILENAME},
     'COOKIES_FILE':             {'default': lambda c: c['COOKIES_FILE'] and Path(c['COOKIES_FILE']).resolve()},
-    'CHROME_USER_DATA_DIR':     {'default': lambda c: find_chrome_data_dir() if c['CHROME_USER_DATA_DIR'] is None else Path(c['CHROME_USER_DATA_DIR']).resolve() or None},
+    'CHROME_USER_DATA_DIR':     {'default': lambda c: find_chrome_data_dir() if c['CHROME_USER_DATA_DIR'] is None else Path(c['CHROME_USER_DATA_DIR']).resolve() if Path(c['CHROME_USER_DATA_DIR']).resolve() != Path.cwd() else None},
```

Comment:
This should work. Another option would be checking against the value without touching paths: `if c['CHROME_USER_DATA_DIR'] else None`.
@pirate which do you prefer?

Reply:
I prefer `if c['CHROME_USER_DATA_DIR'] else None`